lib/goodcity/organisation_populator.rb

```diff
@@ -10,31 +10,29 @@ class OrganisationPopulator
       "website" => "url",
     }
 
-    def initialize
-      begin
-        @file = Nestful.get(URL).response.body
```

Comment:
@trojanh now that we've extracted the URL download out of the initializer there is nothing left to do so it can be removed. Since no initializer is required, we can create a simple `self.run` convenience method so that it's shorter function call i.e. `Goodcity::OrganisationPopulator.run` instead of `Goodcity::OrganisationPopulator.new.run`

Also, the airbrake exceptions should be caught inside the rake task already. If not, then we should add it at the global rake level (for every task) not just this one.